src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC } from 'react';
+
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+
+interface Props {
+  people: Person[];
+  selectedSlug?: string;
+}
+
+const PeopleTable: FC<Props> = ({ people, selectedSlug }) => {
+  const getParent = (parentName: string | null) => {
+    if (!parentName) {
+      return '-';
+    }
+
+    const parent = people.find(person => person.name === parentName);
+
+    if (parent) {
+      return <PersonLink person={parent} />;
+    }
+
+    return parentName;
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => (
+          <tr
+            key={person.slug}
+            data-cy="person"
+            className={
+              selectedSlug === person.slug ? 'has-background-warning' : ''
+            }
```

Comment:
It is better to use libraries like classnames or clsx for conditional classnames